app/renderer/components/preferences/syncTab.js

```diff
@@ -117,7 +118,9 @@ class SyncTab extends ImmutableComponent {
       : getSetting(settings.SYNC_DEVICE_NAME, this.props.settings)
     return <SettingItem>
       <span data-l10n-id='syncDeviceNameInput' />
-      <input className='deviceNameInput formControl' spellCheck='false'
+      <SettingTextbox
```

Comment:
Earlier I tried to use a Textbox however I couldn't get the prop `ref={}` to work so the input didn't work[0]. I gave up and used a standard input.
This branch doesn't seem to set the device name either.
[0] Maybe this: https://facebook.github.io/react/docs/refs-and-the-dom.html

Test Plan:
- Clear profile
- Enable sync and name the device
- Observe terminal for the device name– should not be "browser-laptop"

Also tested by: `yarn test -- --grep='"^Sync Panel"'`